contracts/smart-contract-wallet/utils/GasEstimatorSmartAccount.sol

```diff
@@ -9,15 +9,14 @@ contract GasEstimatorSmartAccount {
     function estimate(
         address _actualWallet,
         address _factory,
-        address _implementation,
-        bytes calldata initializer,
-        uint _index,
+        address _owner,
+        uint256 _index,
         bytes calldata _data // execTransaction data // counterFactual wallet should have assets if required
     ) external returns (bool success, bytes memory result, uint256 gas) {
         // solhint-disable
         uint256 initialGas = gasleft();
         address _wallet = SmartAccountFactory(_factory)
-            .deployCounterFactualWallet(_implementation, initializer, _index);
+            .deployCounterFactualWallet(_owner, _index);
```

Comment:
Do we call this method using eth_call?

Reply:
yes, this contract is for gas estimation test cases when wallet is not already deployed..